### PR TITLE
ESAPI: Uppercase _Async and _Finish and 2 minor additions

### LINKED ISSUE #898: ESAPI: UpperCase _async and _finish

### PR DESCRIPTION
**Pull request by AndreasFuchsSIT**
This PR fixes #898 first and foremost.

It also includes two fixes that we had to bundle in to avoid merge conflicts.

### CODE REVIEW

**Comment by AndreasFuchsSIT** [comment]
I assume you are referring to the second commit 512ac4a7c171b736cb181cd5d7a04c14c387e3ba ESAPI: Fix parameter type in Policies ?

That's one of the two additional fixes that I had to bundle, since they'd be conflicting with this PR if they were a separate one... (And I did not want to wait an extra day)

**Comment by dantpm** [comment]
Yes, OK.

File: src/tss2-esys/api/Esys_NV_UndefineSpaceSpecial.c [hunk 1]
```diff
@@ -212,6 +212,9 @@ TSS2_RC
 Esys_NV_UndefineSpaceSpecial_Finish(
     ESYS_CONTEXT *esysContext)
 {
+     ESYS_TR nvIndex;
```
Comment (flihp) [inline src/tss2-esys/api/Esys_NV_UndefineSpaceSpecial.c]: this looks like 5 spaces to me, should be 4?

Comment (AndreasFuchsSIT) [reply src/tss2-esys/api/Esys_NV_UndefineSpaceSpecial.c]: done

**Pull request merged**